src/Fixer/Phpdoc/NoEmptyPhpdocFixer.php

```diff
@@ -62,8 +63,23 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
-            if (Preg::match('#^/\*\*[\s\*]*\*/$#', $token->getContent())) {
-                $tokens->clearTokenAndMergeSurroundingWhitespace($index);
+            if (!Preg::match('#^/\*\*[\s\*]*\*/$#', $token->getContent())) {
+                continue;
+            }
+
+            $tokens->clearTokenAndMergeSurroundingWhitespace($index);
+
+            $previousToken = $tokens[$index - 1];
+            if ($previousToken->isWhitespace()) {
+                $previousWhitespaceContent = $previousToken->getContent();
+
+                $lastLineBreak = strrpos($previousWhitespaceContent, PHP_EOL);
+                $newWhitespaceContent = substr($previousWhitespaceContent, 0, $lastLineBreak);
+                if ($newWhitespaceContent) {
```

Comment:
is this a test against `false` _or_ `0` by design? I think you want `false !==` here?